app/src/main/java/com/github/multimatum_team/multimatum/activity/AddDeadlineActivity.kt

```diff
@@ -276,24 +323,53 @@ class AddDeadlineActivity : AppCompatActivity() {
             // Reset the text input for future use
             textTitle.text = ""
 
+            // Add the deadline
```

Comment:
same as above, not sur about this comment. You litterally call an addDeadline function.